### PR TITLE
ADD KION DATASET FROM ARTICKLE

### PR DESCRIPTION
**Pull request by irsafilo**

### CODE REVIEW

**Comment by irsafilo** [comment]
add kion dataset from article

File: datasets/KION/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# MTS Kion Implicit Contextualised Sequential Dataset for Movie Recommendation
+
+## Dataset Description
+This is an official repository of the Kion Movies Recommendation Dataset. 
+The data was gathered from the users of [MTS Kion](https://kion.ru/home) video streaming platform from 13.03.2021 to 22.08.2022.
+
+
+The public part of the  dataset includes 5,476,251 interactions of 962,179 users with 15,706 items. The dataset includes user-item interactions; users demographics and rich items meta-data.
```
Comment (blondered) [inline datasets/KION/README.md:8]: includes user_item interactions and their characteristics (such as temporal information, watch duration and watch percentage), as well as user demographics and  rich movies meta-information.

File: datasets/KION/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# MTS Kion Implicit Contextualised Sequential Dataset for Movie Recommendation
+
+## Dataset Description
+This is an official repository of the Kion Movies Recommendation Dataset. 
+The data was gathered from the users of [MTS Kion](https://kion.ru/home) video streaming platform from 13.03.2021 to 22.08.2022.
+
+
+The public part of the  dataset includes 5,476,251 interactions of 962,179 users with 15,706 items. The dataset includes user-item interactions; users demographics and rich items meta-data. 
+
+*Private* part of the dataset contains movies that the users watched within one week following the period covered by the publict dataset. It is not released to general public, however there is a public sandbox, where the researchers can measure MAP@10 metric on the private part of the data. Sandbox is accessible by the address https://ods.ai/competitions/competition-recsys-21/leaderboard/public_sandbox.
```
[the remark this answers is not in the document]
Comment (blondered) [reply datasets/KION/README.md:10]: researchers без the

File: datasets/KION/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Comparison with MovieLens-25M and Netflix datasets
+
+### Quantitative comparison:
+|                              | **Netflix** |**Movielens-25M** | **Kion**           |
+|------------------------------|-------------|------------------|--------------------|
+| Users                        | 480,189     | 162,541          | 962,179            |
+| Items                        | 17,770      | 59,047           | 15,706             |
+| Interactions                 | 100,480,507 | 25,000,095       | 5,476,251          |
+| Avg. Session Length          | 209.25      | 153.80           | 5.69               |
```
Comment (blondered) [inline datasets/KION/README.md:33]: Avg. Sequence Length

**Pull request closed without merging**